checkIn/server/db.js

```diff
@@ -6,7 +6,7 @@ const {
   BOOLEAN,
 } = require('sequelize');
 
-const password = 'secret_Wilson10!';
+const password = 'Richi53!';
```

Comment:
I don't know sequelize very much but committing a variable called password does not feel right, can you double check?

Reply:
Good catch! Thank you